build/teamcity/internal/cockroach/build/ci/build-and-push-bazel-builder-image.sh

```diff
@@ -4,3 +4,21 @@ set -xeuo pipefail
 TAG=$(date +%Y%m%d-%H%M%S)
 docker buildx create --name "builder-$TAG" --use
 docker buildx build --push --platform linux/amd64,linux/arm64 -t "cockroachdb/bazel:$TAG" -t "cockroachdb/bazel:latest-do-not-use" build/bazelbuilder
+
+if [[ "$open_pr_on_success" == "true" ]]; then
+    # Trigger "Open New Bazel Builder Image PR".
+    curl -u "$TC_API_USER:$TC_API_PASSWORD" -X POST \
+      "https://$TC_SERVER_URL/app/rest/buildQueue" \
+      -H 'Accept: application/json' \
+      -H 'Content-Type: application/xml' \
+      -H "Host: $TC_SERVER_URL" \
+      -d '<build branchName="master">
+      <buildType id="Internal_Cockroach_Build_Ci_OpenNewBazelBuilderImagePr"/>
```

Comment:
Does the script this build config calls still need to be added to the repo?

Reply:
It doesn't need to but I put it here because I didn't see anything sensitive about it and thought it's better to have everything at one place, ie: look at the script and know everything this build config does.